package.json

```diff
@@ -151,6 +151,7 @@
     "@types/js-cookie": "2.2.6",
     "@types/lodash": "^4.14.165",
     "@types/node": "^14.14.9",
+    "@types/node-sass": "^4.11.2",
```

Comment:
Why do we need type bindings for node sass in this PR? We never call  `sass` from typescript. I think this may be a mistake.